src/main/java/com/paskar/email/application/repositiory/OwnMapper.java

```diff
@@ -0,0 +1,8 @@
+package com.paskar.email.application.repositiory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OwnMapper extends ObjectMapper {
```

Comment:
This is not a proper way to declare dependency for component from external library. You can either add dependency on spring-boot-autoconfigure module which has JacksonAutoConfiguration that will configure this bean automatically(don't declare version, it will be resolved from parent module) or create configuration class and declare it there:
```
@Configuration
public class MapperConfiguration {
  @Bean
  public ObjectMapper mapper() {
    return new ObjectMapper();
 }
}
```